src/current/_includes/releases/v25.1/upgrade-finalization.md

```diff
@@ -1,4 +1,7 @@
-During a major-version upgrade, certain features and performance improvements may not be available until the upgrade is finalized.
+During a major-version upgrade, certain features and performance improvements may not be available until the upgrade is finalized. In v25.1, these are:
 
-- A cluster must have an [Enterprise license]({% link v24.3/licensing-faqs.md %}#set-a-license) or a [trial license]({% link v24.3/licensing-faqs.md %}#obtain-a-license) set before an upgrade to v24.3 can be finalized.
-- New clusters that are initialized for the first time on v24.3, and clusters that are upgraded to v24.3 will now have a [zone config]({% link v24.3/configure-replication-zones.md %}) defined for the `timeseries` range if it does not already exist, which specifies the value for `gc.ttlseconds`, but inherits all other attributes from the zone config for the `default` range.
+- Support for XA transactions, which allow CockroachDB to participate in distributed transactions with other resources (e.g. databases or message queues) using a two-phase commit protocol. [#129448](https://github.com/cockroachdb/cockroach/pull/129448)
+- Jobs system changes:
+  - `SHOW JOBS` is now based on a new mechanism for storing information about the progress and status of running jobs. [#139230](https://github.com/cockroachdb/cockroach/pull/139230)
+  - `ALTER JOB ... OWNER TO` can now be used to transfer ownership of a job between users/roles. [#138139](https://github.com/cockroachdb/cockroach/pull/138139)
+  - Users can now always see and control (pause/resume/cancel) jobs that they own. [#138178](https://github.com/cockroachdb/cockroach/pull/138178)
```

Comment:
@dt Can you please confirm whether it's correct to include all this in the v25.1 Features Requiring Finalization list, and that the structure under "Jobs system changes" makes sense?